cpp/arcticdb/version/version_store_api.cpp

```diff
@@ -797,6 +797,7 @@ std::vector<std::variant<ReadResult, DataError>> PythonVersionStore::batch_read(
                 }
                 );
     }
+    apply_global_refcounts(handler_data, read_options.output_format());
```

Comment:
Is this and the couple of places in `python_bindings.cpp` the only places where we create Python `None` objects in the native layer?

Reply:
Yes, read methods are affected: `read`, `read_batch`, `batch_read_keys`, `read_index`, `read_column_stats_version`, `LibraryTool::read`